### PR TITLE
Upgrade docs fixes (helm values, remove --force)

### PR DESCRIPTION
**Pull request by bpmct**
Some fixes based on my experience upgrading today.

### CODE REVIEW

File: guides/ssl-certificates/cloudflare.md [hunk 1]
```diff
@@ -116,13 +116,13 @@ spec:
...
-      dnsZones:
-         # Only use this issuer for the domain example.com and its subdomains.
-         - 'example.com'
+      # This section denotes which domains to use this issuer for. If you didn't
+      # limit which zones the API token had access to, you may wish to remove
+      # this section.
+      selector:
+        dnsZones:
+          # Only use this issuer for the domain example.com and its subdomains.
+          - 'example.com'
```
Comment (bpmct) [inline guides/ssl-certificates/cloudflare.md:125]: this is not indented correctly, as discovered when helping someone with their trial. ref: https://cert-manager.io/docs/configuration/acme/#match-labels

File: setup/updating.md [hunk 2]
```diff
@@ -44,13 +44,15 @@ To update Coder, follow these steps:
     helm get values --namespace coder coder > current-values.yml
     ```
 
+    > Double-check your values to ensure it only contains your changes.  
+
```
Comment (bpmct) [inline setup/updating.md:48]: Context: My values file contained references of old 1.16 coder pods when I was attempting to upgrade to 1.17

```sh
➜  cdr-cs git:(main) ✗ helm get values --namespace coder coder > current-values.yml
➜  cdr-cs git:(main) ✗ cat current-values.yml 
USER-SUPPLIED VALUES:
USER-SUPPLIED VALUES: null
cemanager:
  accessURL: ""
  image: docker.io/coderenvs/coder-service:1.16.1
...
```

File: setup/updating.md [hunk 3]
```diff
@@ -44,13 +44,15 @@ To update Coder, follow these steps:
...
 1. Provide your helm chart values file and upgrade to the desired version (e.g.,
 1.16.1):
 
     *Note: If you omit --version, you'll upgrade to the latest version.*
 
     ```bash
-    helm upgrade --namespace coder --force --install --atomic --wait \
+    helm upgrade --namespace coder --install --atomic --wait \
```
Comment (bpmct) [inline setup/updating.md:55]: @sreya said the --force flag causes the helm errors, my install succeeded when I removed it.

**Comment by khorne3** [comment]
@bpmct Does this take care of https://app.clubhouse.io/coder/story/9385/update-docs-can-fail-every-time-forcing-user-to-use-workaround or do you think we should proceed with further edits to make sure that we have an install happy path?

**Review by khorne3 — APPROVED** [review]

**Comment by bpmct** [comment]
> @bpmct Does this take care of https://app.clubhouse.io/coder/story/9385/update-docs-can-fail-every-time-forcing-user-to-use-workaround or do you think we should proceed with further edits to make sure that we have an install happy path?

Yeah, this aims to resolve that issue. Thanks Katie!

I tagged @coadler and @sreya just to make sure I'm not missing anything.

**Review by coadler — APPROVED** [review]

**Pull request merged**